src/index.ts

```diff
@@ -74,7 +66,16 @@ async function main() {
   );
   logger.log("Happy coding!");
 
-  process.exit(0);
+  const repeatApplication =
```

Comment:
This can be simplified: `const repeatApplication = process.env.REPEAT_APPLICATION === 'true';`